src/reynir_correct/wrappers.py

```diff
@@ -261,7 +261,12 @@ def sentence_stream() -> Iterator[List[CorrectToken]]:
     for toklist in sentence_stream():
         len_tokens = len(toklist)
         # Invoke the spelling and grammar checker on the token list
-        sent = check_tokens(toklist)
+        inneroptions: Dict[str, Union[str, bool]] = {}
```

Comment:
Move this out of the loop? No need to do this setup for every sentence.

Reply:
Done